ssl/ssl_locl.h

```diff
@@ -1491,6 +1493,15 @@ typedef struct {
     size_t meths_count;
 } custom_ext_methods;
 
+typedef struct sigalg_lookup_st {
+    const char *name;
+    uint16_t sigalg;
+    int hash;
+    int sig;
+    int sigandhash;
+    int curve;
+} SIGALG_LOOKUP;
```

Comment:
`tls_sigalgs_st` was beautifully commented, and a number of the fields are the same here...  would you mind documenting this one similarly?